packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.ts

```diff
@@ -495,9 +541,7 @@ export class HorizontalScroll extends FoundationElement {
      * @internal
      */
     private getEasedFactor(easing: ScrollEasing, progress: number): number {
-        if (progress > 1) {
-            progress = 1;
-        }
+        progress = progress > 1 ? 1 : progress;
```

Comment:
This works but if you wanted it even more explicit.

`progress = Math.min(1, progress)`

Reply:
Calling `Math.min` is slower than an inline ternary.